homeassistant/components/smartthings/sensor.py

```diff
@@ -156,16 +164,22 @@
 
 async def async_setup_entry(hass, config_entry, async_add_entities):
     """Add binary sensors for a config entry."""
+    from pysmartthings import Capability
     broker = hass.data[DOMAIN][DATA_BROKERS][config_entry.entry_id]
     sensors = []
     for device in broker.devices.values():
         for capability in broker.get_assigned(device.device_id, 'sensor'):
-            maps = CAPABILITY_TO_SENSORS[capability]
-            sensors.extend([
-                SmartThingsSensor(
-                    device, m.attribute, m.name, m.default_unit,
-                    m.device_class)
-                for m in maps])
+            if capability == Capability.three_axis:
+                sensors.extend(
+                    [SmartThingsThreeAxisSensor(
+                        device, index) for index in range(0, 3)])
```

Comment:
Use `range(3)`. Consider iterating `THREE_AXIS_NAMES` instead.

Reply:
I don't care for the element itself at this point, so I changed it to `range(len(THREE_AXIS_NAMES))`